src/components/Section.js

```diff
@@ -0,0 +1,287 @@
+          }
+        >
+          <AddBoxTwoTone
+            style={{
+              color: grey[700],
+            }}
+          />
+          <span
+            style={{
+              color: grey[700],
+            }}
+          >
+            Add to Wishlist
+          </span>
+        </CalendarButton>
+      </UtilBar>
+      <Collapse
+        in={
+          expandedList[section.registrationNumber] === undefined
+            ? false
+            : expandedList[section.registrationNumber]
+        }
+        timeout="auto"
+        unmountOnExit
+      >
+        {section.recitations ? (
+          section.recitations.map((recitation, i) => {
+            let sortedRecitationsMeetings = recitation.meetings
+              ? recitation.meetings.sort(
+                  (a, b) =>
+                    parseDate(a.beginDate).getDay() -
+                    parseDate(b.beginDate).getDay()
+                )
+              : [];
+            return (
+              <Recitation
+                key={i}
+                recitation={recitation}
+                sortedRecitationsMeetings={sortedRecitationsMeetings}
+                courseName={courseData.name}
+                year={year}
+                semester={semester}
+              />
+            );
+          })
+        ) : (
+          <> </>
+        )}
+      </Collapse>
+    </SectionContainer>
+  );
+}
+
+Section.propTypes = {
+  year: PropTypes.number.isRequired,
+  semester: PropTypes.string.isRequired,
+  wishlist: PropTypes.arrayOf(PropTypes.object).isRequired,
+  wishlistCourse: PropTypes.func.isRequired,
+  section: PropTypes.object.isRequired,
+  sortedSectionMeetings: PropTypes.array.isRequired,
+  courseData: PropTypes.object.isRequired,
+};
+
+const SectionContainer = styled.div`
+  padding: 1.8vmin 2.8vmin;
+  background-color: var(--grey100);
+  width: 84%;
+  margin-left: 8%;
+  position: relative;
+
+  & > .sectionName {
+    font-size: 1.8rem;
+    font-family: var(--condensedFont);
+    color: var(--grey800);
+    margin-bottom: 0.25rem;
+  }
+
+  & > .sectionNum {
+    font-size: 1.6rem;
+    font-family: var(--condensedFont);
+    color: var(--grey700);
+    margin: 0 0 -1rem 1rem;
+  }
+
+  & > .attributes {
+    display: flex;
+    flex-wrap: wrap;
+  }
+`;
+
+const ExpandButton = styled.div`
+  font-size: 1.1rem;
+  height: 100%;
+  width: 12rem;
+  border-radius: 0.6rem;
+  padding: 0.8rem 0.5rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+  background-color: ${grey[200]};
+  margin-right: 2rem;
+  transition: 0.1s;
+
+  :hover {
+    background-color: ${grey[300]};
+  }
+
+  & > svg {
+    margin-right: 0.65rem;
+  }
+`;
+
+const SectionDescription = styled.div`
+  padding: 0 1.5rem 1.5rem 0.5rem;
+  max-width: 68%;
+  color: var(--grey700);
+`;
+
+const UtilBar = styled.div`
+  padding: 0.5rem;
+  height: 6vh;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  border-bottom: 1px solid;
```

Comment:
Move this line to SectionContainer's styling. Also implement some way for this line to be hidden on the final section